info: Server started at http://localhost:4000/api/v1/ {"timestamp":"2024-04-11T10:31:56.341Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\index.js:351:10\n    at C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at Database.<anonymous> (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:20:22)\n    at Generator.next (<anonymous>)\n    at C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:8:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:4:12)\n    at Database.initializeDatabase (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:18:57)\n    at App.initializeDatabase (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\index.ts:64:13)\n    at new App (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\index.ts:40:10)\n    at Object.<anonymous> (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\index.ts:92:13)","timestamp":"2024-04-11T10:32:29.632Z"}
info: Server started at http://localhost:4000/api/v1/ {"timestamp":"2024-04-11T10:43:57.449Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\index.js:351:10\n    at C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at Database.<anonymous> (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:20:22)\n    at Generator.next (<anonymous>)\n    at C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:8:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:4:12)\n    at Database.initializeDatabase (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:18:57)\n    at App.initializeDatabase (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\index.ts:64:13)\n    at new App (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\index.ts:40:10)\n    at Object.<anonymous> (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\index.ts:92:13)","timestamp":"2024-04-11T10:44:30.166Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.findOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (C:\Users\USER\Documents\software\catura-api\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/auth/login 
    method - POST 
    IP - ::1
   {"timestamp":"2024-04-11T10:44:40.119Z"}
info: Server started at http://localhost:4000/api/v1/ {"timestamp":"2024-04-11T10:46:08.579Z"}
info: Connected to the database. {"timestamp":"2024-04-11T10:46:12.997Z"}
info: Server started at http://localhost:4000/api/v1/ {"timestamp":"2024-04-11T22:06:10.717Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\index.js:351:10\n    at C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (C:\\Users\\USER\\Documents\\software\\catura-api\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at Database.<anonymous> (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:21:22)\n    at Generator.next (<anonymous>)\n    at C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:8:71\n    at new Promise (<anonymous>)\n    at __awaiter (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:4:12)\n    at Database.initializeDatabase (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\config\\database.ts:19:57)\n    at App.initializeDatabase (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\index.ts:64:13)\n    at new App (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\index.ts:40:10)\n    at Object.<anonymous> (C:\\Users\\USER\\Documents\\software\\catura-api\\src\\index.ts:92:13)","timestamp":"2024-04-11T22:06:43.058Z"}
info: Server started at http://localhost:4000/api/v1/ {"timestamp":"2024-04-11T22:10:55.159Z"}
info: Connected to the database. {"timestamp":"2024-04-11T22:11:08.502Z"}
info: Server started at http://localhost:4000/api/v1/ {"timestamp":"2024-04-11T22:13:59.334Z"}
info: Server started at http://localhost:4000/api/v1/ {"timestamp":"2024-04-11T22:14:07.834Z"}
info: Connected to the database. {"timestamp":"2024-04-11T22:14:35.853Z"}
